models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md

```diff
@@ -0,0 +1,89 @@
+# Brain-tumor-segmentation-0002
+
+## Use Case and High-Level Description
+
+This model was created for participation in the [Brain Tumor Segmentation Challenge](https://www.med.upenn.edu/cbica/brats2019/registration.html) (BraTS) 2019. The code is based on the corresponding paper[TBD](), where the authors employ knowledge distillation for automatic brain tumor segmentation. First, they annotated unlabeled data with ensemble of different models. Then, they trained simple UNet model with residual blocks.
```

Comment:
Presumably you're going to fill in the paper later?

Reply:
Right

Reply:
Then leave the conversation open until you do. :-)

Reply:
We need to add this model in R4 while the paper can be submitted later. I'd propose to merge with such message or I can remove any notes about the article for the moment

Reply:
I think it should be enough to say that model was created to participate challenge and it have unet arch trained on residual blocks.